src/frontend/components/VideoPlayer/index.tsx

```diff
@@ -115,6 +115,7 @@ const BaseVideoPlayer = ({
         ref={videoNodeRef}
         crossOrigin="anonymous"
         poster={thumbnailUrls[720]}
+        tabIndex={-1}
```

Comment:
add a comment or better a test with explanation so it doesn't disappear in a future update?